app/styles/signup.css

```diff
@@ -78,4 +82,87 @@ h1 {
 .help-msg {
   font-size: 10px;
   text-align: left;
-}
\ No newline at end of file
+}
+
+/* New Signup flow styles */
+
+.container {
+  width: 100%;
+  color: #666;
```

Comment:
Can we please use CSS variables here?
Please follow the pattern in `variables.css` file